README.md

```diff
@@ -5,27 +5,52 @@
 
-- [Quick Start](https://github.com/microsoft/react-native-test-app/wiki/Quick-Start)
-- [Configuring the Test App](#configuring-the-test-app)
+- [Quick Start](#quick-start-⚡)
 - [Migrate an Existing Test App](https://github.com/microsoft/react-native-test-app/wiki/Migrate-an-Existing-Test-App)
-- [Known Issues](#known-issues)
+- [Configuring the Test App](https://github.com/microsoft/react-native-test-app/wiki/Manifest-%28app.json%29)
+- [Known Issues](#known-issues-⚠️)
 
 React Native Test App (RNTA) provides test apps for all platforms as a package.
 It handles the native bits for you so you can focus on what's important: your
 product.
 
-If you want to learn how RNTA is used at Microsoft, and see a demo of how to add
-it to an existing library - you can watch the
-["Improve all the repos – exploring Microsoft’s DevExp"](https://youtu.be/DAEnPV78rQc?t=499)
-talk by [@kelset](https://github.com/kelset) and
-[@tido64](https://github.com/tido64) from React Native Europe 2021.
+## Libraries Using RNTA 📚
 
-In the wiki, you can read more about
-[the motivation](https://github.com/microsoft/react-native-test-app/wiki#motivation)
-and [the design](https://github.com/microsoft/react-native-test-app/wiki/Design)
-of this tool.
+<table>
+<tr>
+<td>
 
-## Quick Start
+- Microsoft's
+  [fluentui-react-native](https://github.com/microsoft/fluentui-react-native)
+- Microsoft's [rnx-kit](https://github.com/microsoft/rnx-kit)
+- Shopify's [restyle](https://github.com/Shopify/restyle)
+- [react-native-async-storage](https://github.com/react-native-async-storage/async-storage)
+- [react-native-datetimepicker](https://github.com/react-native-datetimepicker/datetimepicker)
+- [react-native-google-signin](https://github.com/react-native-google-signin/google-signin)
+- [react-native-masked-view](https://github.com/react-native-masked-view/masked-view)
+- [react-native-menu](https://github.com/react-native-menu/menu)
+
+</td>
+<td>
+
+- [react-native-add-calendar-event](https://github.com/vonovak/react-native-add-calendar-event)
+- [react-native-apple-authentication](https://github.com/invertase/react-native-apple-authentication)
+- [react-native-blur](https://github.com/Kureev/react-native-blur)
+- [react-native-image-editor](https://github.com/callstack/react-native-image-editor)
+- [react-native-netinfo](https://github.com/react-native-netinfo/react-native-netinfo)
+- [react-native-webview](https://github.com/react-native-webview/react-native-webview)
+- [react-native-segmented-control](https://github.com/react-native-segmented-control/segmented-control)
+- [sparkfabrik-react-native-idfa-aaid](https://github.com/sparkfabrik/sparkfabrik-react-native-idfa-aaid)
+
+</td>
+</tr>
+</table>
+
+...[and many more!](https://github.com/microsoft/react-native-test-app/network/dependents)
+
+_Are you using `react-native-test-app`? Submit a PR to add it to the list!_
```

Comment:
Let's be consistent about using RNTA:

```suggestion
_Are you using RNTA? Submit a PR to add it to the list!_
```

In case we rename it to `react-native-template-app` later 😉